python/lsst/ts/xml/data/sal_interfaces/ESS/ESS_Telemetry.xml

```diff
@@ -625,4 +625,372 @@
       <Count>1</Count>
     </item>
   </SALTelemetry>
+  <SALTelemetry>
+    <EFDB_Topic>ESS_earthquakeBH</EFDB_Topic>
```

Comment:
I just realized what the names of these topics means. I would suggest you also expand these names. It would make much easier for us to find the information in the EFD. Consider something like `ESS_earthquakeBroadBandHighGain` and so on.